.github/api-extractor-action/api-extractor.json

```diff
@@ -307,6 +307,11 @@
       "default": {
         "logLevel": "warning"
         // "addToApiReportFile": false
+      },
+
+      // Spartaus: currently we don't use the release tags @alpha/@beta/@internal/@public in our JSDocs:
+      "ae-missing-release-tag": {
+        "logLevel": "none"
       }
 
       // "ae-extra-release-tag": {
```

Comment:
In the report, there exist `ae-forgotten-export`, but I don't see a config option to disable it https://api-extractor.com/pages/messages/ae-forgotten-export/. 

Moreover, I don't see a reason for why it would complain when I do see those models/interfaces/enums are being exported. 

Examples can be found from the report [here](https://github.com/SAP/spartacus/pull/15908)

Reply:
Not sure what to do with those warnings to be honest.

Reply:
Those warnings are not helpful also because the `OrderEntry` for example IS exported, but from a different library. So the warning is false-positive in this case. Perhaps it's due to our misconfiguration or misusage of the API Extractor tool?

Getting notified about potentially forgotten exports is not a primary purpose of our Public API Changes bot. So I don't want invest time into investigating if we could possibly get rid of false positives here.

Although getting hints like “You forgot to export something” would be nice nice…. if doens’t work correctly ootb in our case, I'm for disabling this warning.